app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -1,333 +1,264 @@
-  ol, ul { list-style: none }
+input[type=text],
+input[type=password],
+input[type=email],
+input[type=number],
+input[type=url],
+input[type=tel],
+textarea {
+  background-color: $input-background-color;
+  border: 1px solid $input-border-color;
+  border-radius: $input-border-radius;
+  color: $input-color;
+  display: inline-block;
+  outline: none;
+  padding: $padding-base-vertical $padding-base-horizontal;
+  transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
+  width: calc(80% - #{($padding-base-horizontal * 2) + 2px});
+
+  &:focus {
+    border: 1px solid $input-border-focus-color;
+    box-shadow: 0 0 4px $input-border-focus-color;
+  }
+}
+
+.formtastic {
+  ol, ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .input {
+    margin-bottom: 15px;
+  }
+
+  .label {
+    display: block;
+    float: left;
+    font-weight: $font-weight-bold;
+    width: 20%;
+
+    abbr {
+      border: none;
+      color:  $required-field-marker-color;
+      margin-left: 0.3em;
+    }
+  }
+
+  .inputs {
+    background-color: $panel-background-color;
+    border-radius: $panel-border-radius;
+
+    > ol {
+      padding: 15px;
+    }
+  }
 
   fieldset {
     border: 0;
-    padding: 10px 0;
-    margin-bottom: 20px;
-
-    &.inputs { @include section-background; }
+    margin: 0 0 20px 0;
+    padding: 0;
+    position: relative; // for abosolute positioning sub fieldset legends for choices/fragments
 
     legend {
       width: 100%;
-      span { display: block; @include section-header; }
-    }
 
-    ol > li {
-      padding: 10px;
-      label {
+      span {
+        background-color: $panel-heading-background-color;
+        border-radius: $panel-border-radius $panel-border-radius 0 0;
         display: block;
-        width: 20%;
-        float: left;
-        font-size: 1.0em;
-        font-weight: bold;
-        color: $form-label-color;
-        abbr {
-          border: none;
-          color:  $required-field-marker-color;
-        }
+        font-weight: $font-weight-bold;
+        padding: $panel-heading-padding;
       }
     }
 
-    ol > li.has_many_container {
-      padding: 20px 10px;
+    .has_many_container {
       h3 {
-        font-size: 12px;
-        font-weight: bold;
-      }
-      .has_many_fields { margin: 10px 0 }
-    }
-
-    ol > li > li label {
-      line-height:100%;
-      padding-top:0;
-      input {
-        line-height:100%;
-        vertical-align:middle;
-        margin-top:-0.1em;
+        border-bottom: 1px solid #e8e8e8;
+        margin-bottom: 20px;
       }
     }
   }
 
   // relative so the absolutely-positioned handle will fit
-  .has_many_fields { position: relative }
-
-  .has_many_container {
-    .handle {
-      position: absolute;
-      top: calc(50% - 3em / 2);
-      right: 2px;
-      padding: 0;
-      cursor: move;
-    }
+  .has_many_fields {
+    border-bottom: 1px solid #e8e8e8;
```

Comment:
Color literals like `#e8e8e8` should only be used in variable declarations; they should be referred to via variable everywhere else.